catalog/core/catalog-core-commands/pom.xml

```diff
@@ -113,7 +113,7 @@
         <dependency>
             <groupId>net.lingala.zip4j</groupId>
             <artifactId>zip4j</artifactId>
-            <version>1.3.2</version>
+            <version>${zip4j.version}</version>
```

Comment:
@mcalcote Keep an eye on this PR. Whether you or @Kjames5269 get in first will make a difference here: https://github.com/codice/ddf/pull/3711

Reply:
Thanks for the heads up 👍